version.go

```diff
@@ -0,0 +1,22 @@
+package triton
+
+import (
+	"fmt"
+	"runtime"
+)
+
+// The main version number of the current released Triton-go SDK.
+const Version = "1.0.0"
```

Comment:
Can we jump to `"0.9.0"` for a week before calling it `1.0.0` so that we can integrate this into other tools and make sure we don't need to make any other tweaks?